app/src/main/kotlin/PushYourLuck/PushYourLuckState.kt

```diff
@@ -0,0 +1,6 @@
+package PushYourLuck
+
+open class PushYourLuckState(diceMarkedSides: MutableList<MutableList<Boolean>>) {
+    var state = diceMarkedSides
```

Comment:
```suggestion
open class PushYourLuckState(var state: MutableList<MutableList<Boolean>>) {
```
you don't need to declare a class variable separately from the primary constructor

Reply:
@JunTaoLuo its been updated to address this.